tests/e2e/specs/wcpay/merchant/merchant-orders-partial-refund.spec.js

```diff
@@ -48,167 +48,154 @@ const dataTable = [
 	],
 ];
 
-// TODO: Unskip (remove outer block) once refund E2E tests failure are investigated.
-describe.skip( 'Partial refund tests', function () {
-	describe.each( dataTable )(
-		'Order > Partial refund',
-		( testName, { lineItems, refundInputs } ) => {
-			let orderId;
-			let orderTotal;
-
-			beforeAll( async () => {
-				// Set up the test order
-				await setupProductCheckout(
-					config.get( 'addresses.customer.billing' ),
-					lineItems
-				);
-				await fillCardDetails( page, card );
-				await shopper.placeOrder();
-				await expect( page ).toMatch( 'Order received' );
-
-				// Remember the order ID and order total. We will need them later.
-				orderId = await page.$eval(
-					'.woocommerce-order-overview__order.order > strong',
-					( el ) => el.innerText
-				);
-				orderTotal = await page.$eval(
-					'.woocommerce-order-overview__total strong',
-					( el ) => Number( el.innerText.replace( '$', '' ) )
-				);
-
-				// Login as merchant
-				await merchant.login();
-			}, 200000 );
-
-			afterAll( async () => {
-				await merchant.logout();
-			} );
-
-			it( `should refund ${ testName }`, async () => {
-				const refundReason = `Refunding ${ testName }`;
-				const refundTotal = refundInputs
-					.map( ( { refundAmount } ) => refundAmount )
-					.reduce( ( acc, cur ) => acc + cur );
-				const refundTotalString = refundTotal.toFixed( 2 );
-				const netPayment = ( orderTotal - refundTotal ).toFixed( 2 );
-
-				await merchant.goToOrder( orderId );
-
-				// We need to remove any listeners on the `dialog` event otherwise we can't catch the dialog below
-				await page.removeAllListeners( 'dialog' );
-
-				// Click the Refund button
-				await expect( page ).toClick( 'button.refund-items' );
-
-				// Fill up the quantity and/or amount to be refunded per line item
-				const rows = await page.$$( '#order_line_items tr.item' );
-				for ( let i = 0; i < refundInputs.length; i++ ) {
-					const { refundQty, refundAmount } = refundInputs[ i ];
-					const row = rows[ i ];
-
-					if ( refundQty ) {
-						await expect( row ).toFill(
-							'.refund_order_item_qty',
-							`${ refundQty }`
-						);
-					} else {
-						await expect( row ).toFill(
-							'.refund_line_total',
-							`${ refundAmount }`
-						);
-					}
+describe.each( dataTable )(
+	'Order > Partial refund',
+	( testName, { lineItems, refundInputs } ) => {
+		let orderId;
+		let orderTotal;
+
+		beforeAll( async () => {
+			// Disable multi currency in the merchant settings.
+			await merchant.login();
+			await merchantWCP.openWCPSettings();
```

Comment:
Do you think moving these to a separate utility method for reusing the multi-currency disabling could be helpful?

Reply:
Indeed, I'll write utility method for it, thanks!

Reply:
@deepakpathania added your suggestion in https://github.com/Automattic/woocommerce-payments/pull/7334/commits/6c2975a9b8a995aa15c1bf4f45a087dc326e3128 thanks